### ISSUE TITLE
Use of Revision::getTimestampFromId was deprecated in MediaWiki 1.35. [Called from SMWExportController::printPages

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SemanticMediaWiki#4714]
### Setup and configuration

```
Using PHP 7.4.0

Semantic MediaWiki: 3.2.0-alpha, git: 5a80f51
                    SMWSQLStore (mysql:5.6.33-0ubuntu0.14.04.1)

MediaWiki:          1.35.0-alpha, git: 24b1aae (refs/heads/master)
                    Extension vendor autoloader

Site language:      en
Execution time:     2020-04-11 01:45

Debug logs:         Disabled
Xdebug:             Disabled (or not installed)

Intl/ICU:           7.4.0 / 52.1
PCRE:               10.33 2019-04-16
```

### Issue

```
2) SMW\Tests\Integration\InterwikiDBIntegrationTest::testRdfSerializationForInterwikiAnnotation

Use of Revision::getTimestampFromId was deprecated in MediaWiki 1.35. [Called from SMWExportController::printPages in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/export/SMW_ExportController.php at line 468]

/home/travis/build/SemanticMediaWiki/mw/includes/debug/MWDebug.php:333
/home/travis/build/SemanticMediaWiki/mw/includes/debug/MWDebug.php:265
/home/travis/build/SemanticMediaWiki/mw/includes/GlobalFunctions.php:1033
/home/travis/build/SemanticMediaWiki/mw/includes/Revision.php:1076
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/export/SMW_ExportController.php:468
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/Integration/InterwikiDBIntegrationTest.php:154
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/Integration/InterwikiDBIntegrationTest.php:91
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/DatabaseTestCase.php:173
/home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:99
```

https://travis-ci.org/github/SemanticMediaWiki/SemanticMediaWiki/jobs/673755527

**Comment by mwjames** [comment]
We should kick `Revision::getTimestampFromId` (and its filtering) from the controller considering "... dropping the $revisiondate filtering and all associated functionality ... " because constructing a revision service is a pain and pulls in unnecessary dependencies that are not worth the effort especially given the overall complexity of `SMWExportController`.

https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/935a6ebd361256b64078e4d0c59a03d4c8e8bef0/includes/export/SMW_ExportController.php#L452-L468

**Comment by Abijeet** [comment]
This was fixed in bf171f6391f85516c07acb03a4815183c5f9733f
